services/sn_analysis_service/app.py

```diff
@@ -28,33 +31,51 @@
 default_analysis_parameters = {"source": "nugent-sn2p", "fix_z": False}
 
 
-def upload_analysis_results(results, data_dict):
+def upload_analysis_results(results, data_dict, request_timeout=60):
```

Comment:
Are there also size limits on these posts? I guess the same size as for all handlers?

Reply:
I'm not limiting the size. baselayer/nginx limits the size to 10M via `client_max_body_size`: 
```
baselayer/services/nginx/nginx.conf.template:
    client_max_body_size 10M;
```
In principle we could make this larger just for `/api/webhook` endpoints. But I think 10M is a good size.